app/controllers/api/v1/featured_tags_controller.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+class Api::V1::FeaturedTagsController < Api::BaseController
+  before_action -> { doorkeeper_authorize! :read, :'read:featured_tags' }, only: [:index, :show]
+  before_action -> { doorkeeper_authorize! :write, :'write:featured_tags' }, except: [:index, :show]
+
+  before_action :require_user!
+  before_action :set_featured_tags, only: :index
+  before_action :set_featured_tag, except: [:index, :create]
+
+  def index
+    render json: @featured_tags, each_serializer: REST::FeaturedTagSerializer
+  end
+
+  def show
+    render json: @featured_tag, serializer: REST::FeaturedTagSerializer
+  end
+
+  def create
+    @featured_tag = current_account.featured_tags.new(featured_tag_params)
+    @featured_tag.reset_data
+    @featured_tag.save
+    render json: @featured_tag, serializer: REST::FeaturedTagSerializer
+  end
+
+  def update
+    @featured_tag.update!(featured_tag_params)
+    render json: @featured_tag, serializer: REST::FeaturedTagSerializer
+  end
+
+  def destroy
+    @featured_tag.destroy!
+    render_empty
+  end
+
+  private
+
+  def set_featured_tag
+    @featured_tag = current_account.featured_tags.find(params[:id])
+  end
+
+  def set_featured_tags
+    @featured_tags = current_account.featured_tags.order(statuses_count: :desc).reject(&:new_record?)
```

Comment:
I don't think the `reject` is necessary? Nothing I see would add unsaved records here

Reply:
This was not necessary. Remove.